# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sphinx-rtd-theme/hotfix-setup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- sphinx_rtd_theme-3.1.0/setup.py.vanilla	2026-09-27 15:13:09.528363516 +0300
+++ sphinx_rtd_theme-3.1.0/setup.py	2026-09-27 15:18:03.565380162 +0300
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-import distutils.cmd
+from setuptools._distutils.cmd import Command
 import os
 import subprocess
 from io import open
@@ -8,7 +8,7 @@
 from setuptools import setup
 
 
-class WebpackBuildCommand(distutils.cmd.Command):
+class WebpackBuildCommand(Command):
 
     description = "Generate static assets"
 
@@ -26,7 +26,7 @@
             subprocess.run(['node_modules/.bin/webpack', '--config', 'webpack.prod.js'], check=True)
 
 
-class WebpackDevelopCommand(distutils.cmd.Command):
+class WebpackDevelopCommand(Command):
 
     description = "Run Webpack dev server"
 
@@ -45,7 +45,7 @@
         )
 
 
-class UpdateTranslationsCommand(distutils.cmd.Command):
+class UpdateTranslationsCommand(Command):
 
     description = "Run all localization commands"
 
@@ -68,7 +68,7 @@
             self.run_command(cmd_name)
 
 
-class TransifexCommand(distutils.cmd.Command):
+class TransifexCommand(Command):
 
     description = "Update translation files through Transifex"
 
